pyupgrade.py

```diff
@@ -1103,9 +1116,13 @@ def __init__(self):  # type: () -> None
         self.if_py3_blocks = set()  # type: Set[ast.If]
 
         self.native_literals = set()  # type: Set[Offset]
-        self.simple_ids = {}  # type: Dict[Offset, ast.Name]
 
-        self._six_from_imports = set()  # type: Set[str]
+        self._from_imported_names = collections.defaultdict(
+            set,
+        )  # type: DefaultDict[str, Set[str]]
```

Comment:
I've just been using `Dict` for this alias -- works well enough and then you don't need a special typing import for it (and we don't use any of the apis of `DefaultDict`)

Reply:
sure